src/Cpu.re

```diff
@@ -65,6 +65,10 @@ let reset = cpu => {
   cpu.pc = Memory.get_word(cpu.memory, cpu.pc);
 };
 
+let now = cpu => {
+  Disassemble.make(cpu.memory)(cpu.pc, 1);
```

Comment:
Just curious, does `Disassemble.make(cpu.memory, cpu.pc, 1)` work? Or do you have to break those up?

Reply:
I hadn't even tried it! Intuitively, it's not obvious to me that when I call a function that returns a function that I can pass all the arguments to both up front. I'd love to discuss that aspect of the evaluation model with you to understand better. 🤘

Reply:
I think it's Haskellish in the sense that every function is always fully curried and always takes a single argument. `f(a, b, c)` is just syntactic sugar for `f(a)(b)(c)` - https://reasonml.github.io/docs/en/function#currying.